grakn-test/test-snb/src/validate-snb/readwrite_grakn--ldbc_driver_config--db_validation.properties

```diff
@@ -0,0 +1,164 @@
+name=LDBC-SNB
+
+# path specifying where to write the benchmark results file
+# STRING
+# COMMAND: -rd/--results_dir
+results_dir=LDBC_SNB_RESULTS
+
+# create a csv file containing simple data about the execution of every operation in the workload
+# BOOLEAN
+# COMMAND: -rl/--results_log
+results_log=true
+
+# time unit to use for measuring performance metrics (e.g., query response time)
+# ENUM ([NANOSECONDS, MICROSECONDS, MILLISECONDS, SECONDS, MINUTES])
+# COMMAND: -tu/--time_unit
+time_unit=MILLISECONDS
+
+# used to 'compress'/'stretch' durations between operation start times to increase/decrease benchmark load
+# e.g. 2.0 = run benchmark 2x slower, 0.1 = run benchmark 10x faster
+# DOUBLE
+# COMMAND: -tcr/--time_compression_ratio
+time_compression_ratio=1.0
+
+# NOT USED AT PRESENT - reserved for distributed driver mode
+# specifies the addresses of other driver processes, so they can find each other
+# LIST (e.g., peer1|peer2|peer3)
+# COMMAND: -pids/--peer_identifiers
+peer_identifiers=
+
+# generate validation parameters file for validating correctness of database implementations
+# parameter values specify: (1) where to create the validation parameters file (2) how many validation parameters to generate
+# STRING|INTEGER (e.g., validation_parameters.csv|1000)
+# COMMAND: -cvp/--create_validation_parameters
+# create_validation_parameters=
+
+# enable validation that will check if the provided workload implementation is correct
+# BOOLEAN
+# COMMAND: -vw/--validate_workload
+validate_workload=true
+
+# calculate & display workload statistics (operation mix, etc.)
+# BOOLEAN
+# COMMAND: -stats/--workload_statistics
+workload_statistics=false
+
+# sleep duration (ms) injected into busy wait loops (to reduce CPU consumption)
+# LONG (milliseconds)
+# COMMAND: -sw/--spinner_wait_duration
+spinner_wait_duration=0
+
+# print help string - usage instructions
+# BOOLEAN
+# COMMAND: -help
+help=false
+
+# executes operations as fast as possible, ignoring their scheduled start times
+# BOOLEAN
+# COMMAND: -ignore_scheduled_start_times
+ignore_scheduled_start_times=false
+
+# ***************************************************************
+# *** the following should be set by workload implementations ***
+# ***************************************************************
+
+# fully qualified class name of the Workload (class) implementation to execute
+# STRING (e.g., com.ldbc.driver.workloads.ldbc.snb.interactive.LdbcSnbInteractiveWorkload)
+# COMMAND: -w/--workload
+workload=com.ldbc.driver.workloads.ldbc.snb.interactive.LdbcSnbInteractiveWorkload
+
+# number of operations to generate during benchmark execution
+# LONG
+# COMMAND: -oc/--operation_count
+operation_count=100000
+
+# ************************************************************************************
+# *** non-core configuration parameters ***
+# ************************************************************************************
+
+ldbc.snb.interactive.LdbcQuery10_enable=true
```

Comment:
helpfully, no...  Those are the queries it wants to try. The ones we actually are supporting are registered in the class GraknDb